person.rb

```diff
@@ -20,15 +21,13 @@ def validate_name
     @name = @corrector.correct_name @name
   end
 
+  def add_rental(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] To achieve the 'has-many' association, the person itself should be sent as a parameter to create the new rental.
The code should look similar to this:
```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
 end
```

Reply:
Hello @oloomoses, thanks for your review! Implemented.

Reply:
Hello @oloomoses, thanks for your review! Implemented.